cmd/podman/containers/ps.go

```diff
@@ -173,17 +173,29 @@ func quietOut(responses []entities.ListContainer) error {
 }
 
 func getResponses() ([]entities.ListContainer, error) {
-	responses, err := registry.ContainerEngine().ContainerList(registry.GetContext(), listOpts)
-	if err != nil {
-		return nil, err
+	list := make([]entities.ListContainer, 0)
+	var err error
+
+	for waiting := true; waiting; {
+		list, err = registry.ContainerEngine().ContainerList(registry.GetContext(), listOpts)
+		switch {
+		case err == nil:
+			waiting = false
+		case strings.Contains(err.Error(), "no such container"):
+			logrus.Debugf(err.Error())
+			waiting = true
+		default:
+			return nil, err
+		}
 	}
```

Comment:
Any chance this could loop for a really long time?  I just wonder if there should be a failsafe loop breaker here...  Or maybe it's too late to be reviewing golang black magic?